ci/rpm7/build-dep.sh

```diff
@@ -35,7 +35,6 @@ yum -y install \
   openssl-devel \
   libuuid-devel \
   make \
-  mongodb-org \
```

Comment:
In order to effectively remove mongo from the CI image I think this is not enough. You should:

* Review build.sh script looking for ocurrences of `mongod`
* Remove mongodb.repo file (including the `ADD mongodb.repo /etc/yum.repos.d/` lline in ci/rpm7/Dockerfile)

Reply:
Fixed in d0b12f5